docs/reuse/apps/app-install.md

```diff
@@ -1,18 +1,20 @@
 To install the app:
 
 1. Select **App Catalog**.
-1. In the **Search Apps** field, search for and then select your app.
-1. Optionally, you can scroll down to preview the dashboards included with the app. Then, click **Install App** (sometimes this button says **Add Integration**).
+1. In the 🔎 **Search Apps** field, run a search for your desired app, then select it.
+1. Click **Install App**.
     :::note
-    If your app has multiple versions, you'll need to select the version of the service you're using before installation.
```

Comment:
Why are we updating this note?

Reply:
Hi, @samyakjain0606. Just sent you a Slack message re: the historical background of this note.